roles/postgres/tasks/apt_repo.yml

```diff
@@ -0,0 +1,16 @@
+---
+# Ansible module apt_repository rely on unmaintained python libs so using template instead
+- name: Install PostgreSQL deb repository
+  become: true
+  template:
+    src: apt_repo.j2
```

Comment:
Why not use apt_repository?
https://docs.ansible.com/ansible/2.9/modules/apt_repository_module.html

Reply:
Actually I tried to but failed due to some dependencies that were not playing well with python3.
IIRC the main problem I had was with python-distutils-extra